docs/h_dict.md

```diff
@@ -82,3 +83,18 @@ The `h_dict` is a Python dictionary that contains all the configurations for eac
 | `costs` | dict | Cost parameters |
 | `cell_params` | dict | Cell parameters |
 | `degradation` | dict | Degradation parameters |
+
+### External Data (`external_data`)
+| Key | Type | Description | Default |
+|-----|------|-------------|---------|
+| `external_data_file` | str | Path to CSV file with external time series data | Required |
```

Comment:
Is this Required? Some simulations don't need external data files? Perhaps I'm not fully understanding

Reply:
Ah, interesting. So, having the top-level `external_data` field with nothing in it is not allowed, is that correct? I wonder if there is a protocol here; it seems that there might be cases when it's nice to be able to have "empty" top levels (say, an empty `wind_farm`), just as placeholders; but if I'm understanding correctly, that is not currently supported. In that case, I see your point, and I suppose this is fine (Required if `external_data` is provided at all)